main.py

```diff
@@ -35,5 +36,5 @@
 
         bq.create_table('{}.{}.{}'.format(args.project, args.dataset, table), bq_schema, args.dry_run)
 
-        # TODO: columns -> select statement
+        select_statement = generate_select_statement(table, bq_schema, args.salt)
```

Comment:
It needs to be select statement to MySQL DB. So path `schema` instead of `bq_schema`